api/api.proto

```diff
@@ -1883,15 +1883,27 @@ message SdkCloudBackupDeleteAllRequest {
 message SdkCloudBackupDeleteAllResponse {
 }
 
-// Defines a request to list the backups stored by a cloud provider
+// Defines a request to list the backups stored by a cloud provider.
+// The following combinations can be used to get cloud backup information:
+//
+// * For a specific volume in current cluster: Set `src_volume_id` to your desired volume id
+// and do not provide `cluster_id` and `all`.
+// * For a specific volume in a specific cluster: Set `src_volume_id` to your desired volume id
+// and specify `cluster_id`.
+// * For a specific volume in all clusteres: Set `src_volume_id` to your desired volume id
+// and specify set `all` to true, do not provide `cluster_id`.
+// * For all volumes in current cluster: do not provide `cluster_id`, `volume_id` and `all`.
```

Comment:
'and all' or 'or all'?

Reply:
i did not want them to think they shouldn't provide volume_id OR 'all' they shouldn't provide any